.github/workflows/ci.yml

```diff
@@ -458,6 +458,14 @@ jobs:
         if: matrix.modules == 'plugin/trino-bigquery' && env.BIGQUERY_CASE_INSENSITIVE_CREDENTIALS_KEY != ''
         run: |
           $MAVEN test ${MAVEN_TEST} -pl :trino-bigquery -Pcloud-tests-case-insensitive-mapping -Dbigquery.credentials-key="${BIGQUERY_CASE_INSENSITIVE_CREDENTIALS_KEY}"
+      - name: Iceberg Glue Catalog Tests
```

Comment:
@nineinchnick can you take a look at this? This step didn't run on the CI where I was expecting it to

https://github.com/trinodb/trino/runs/5423189712?check_suite_focus=true

Reply:
Ah, I think I see it. `env.AWS_ACCESSKEY` should be `env.AWS_ACCESS_KEY_ID`